runtime/error.c

```diff
@@ -1,12 +1,103 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
 #include "balrt.h"
+#include "third-party/libbacktrace/backtrace.h"
+
+#define INITIAL_PC_COUNT 1
+#define MAX_PC_COUNT 256
+#define SKIP_FROM_END 1 // This number counts from the last stack frame
+#define THREAD 0
+
+struct backtrace_state *state;
+
+typedef struct {
+    uint32_t npcs;
+    uint32_t szpcs;
+    PC *pcs;
+} Trace;
```

Comment:
This is just a vector of PCs.  So call it PCVector or something like that.

Reply:
Changed